web/src/components/pages/dashboard/challenge/achievement.ts

```diff
@@ -0,0 +1,69 @@
+export type AchievementType =
```

Comment:
We typically name our types without a -`Type` postfix. Let's rename this to `Achievement` for consistency.

Reply:
I think `achievement.` is an abstract concept. The other developers won't know what does `achievement` means at first look. So I add postfix `Type` to tell others this only a type.  And I checked the files under the `web` folder. Most type add `type` postfix. such as `HeroType`, `avatarType` and so on.